contracts/Core/StakeManager.sol

```diff
@@ -83,20 +142,14 @@ contract StakeManager is Utils, ACL, StakeStorage {
             stakerId = numStakers;
             stakerIds[msg.sender] = stakerId;
         } else {
-            //WARNING: ALLOWING STAKE TO BE ADDEDD AFTER WITHDRAW/SLASH. consequences unknown
```

Comment:
> WARNING: Potential security vulnerability. Could increase stake maliciously

I removed this comment because `//WARNING: unchecked underflow` is commented below this one, so since we have upgraded `solc` to `v0.8.0` arithmetic underflow/overflow is taken care of.

This comment can be restored, but we should create a task to check whether there's vulnerability or not, if there is we should fix it.  @hrishikeshio